testing/endpoint.go

```diff
@@ -423,21 +423,32 @@ func (endpoint *Endpoint) ChanCloseInit() error {
 
 // SendPacket sends a packet through the channel keeper using the associated endpoint
 // The counterparty client is updated so proofs can be sent to the counterparty chain.
-func (endpoint *Endpoint) SendPacket(packet exported.PacketI) error {
-	channelCap := endpoint.Chain.GetChannelCapability(packet.GetSourcePort(), packet.GetSourceChannel())
+// The packet sequence generated for the packet to be sent is returned. An error
+// is returned if one occurs.
+func (endpoint *Endpoint) SendPacket(
+	sourcePort string,
+	sourceChannel string,
```

Comment:
These can be obtained from the endpoint, `endpoint.ChannelID`, `endpoint.ChannelConfig().PortID`

Reply:
Sorry, I realize now what you mean. I will make the changes.